src/commands/showReferrer.ts

```diff
@@ -0,0 +1,88 @@
+import * as vscode from 'vscode';
+import { IActionContext } from '@microsoft/vscode-azext-utils';
+import { execAsync } from '../support/execAsync';
+import { window } from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import { access } from 'fs/promises'; 
+
+const errorMessage = 'ORAS not found on default path. Download ORAS or update path: ';
+const goToOrasButton = 'Download ORAS';
+const laterButton = 'Later';
+const validTag = /^[a-z0-9./_:-]+$/; 
+
+// check oras exist on user computer
+async function getOrasPath(path: any, bool : boolean): Promise<[string, boolean]> {
```

Comment:
Nit: Specify what the return values are here in a comment. Also, the comment in question is grammatically off and only notes part of what this function does